pyfar/classes/coordinates.py

```diff
@@ -2,7 +2,7 @@
 The following documents the pyfar coordinates class and functions for
 coordinate conversion. More background information is given in
 :py:mod:`coordinates concepts <pyfar._concepts.coordinates>`.
-Available sampling schemes are listed at :py:mod:`~pyfar.samplings`.
+Available sampling schemes are listed at :py:mod:`~spharpy.samplings`.
```

Comment:
For external package documentations to be linked, you'll need to specify these more explicitly.
There's the `intersphinx` plug-in for this. I've used it in spharpy to reference the Pyfar docs there. You can have a look at the respective config in spharpy

for example in spharpy, I used 
:doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>`
to reference the pyfar Coordinates.